src/engine/strat_engine/tests/util.rs

```diff
@@ -0,0 +1,70 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+use std::path::PathBuf;
+
+use devicemapper::{DM, DmFlags, DevId};
+
+use nix::mount::umount;
+use mnt::get_submounts;
+
+/// Remove all device mapper devices which have a name which matches the naming convention for
+/// a stratis created device.
+fn dm_stratis_devices_remove() {
+    match DM::new() {
+        Ok(dm) => {
```

Comment:
While we never unwrap in source code, we unwrap everywhere both in the test code and in the test infrastructure. We could reconsider the policy, but we shouldn't do it piecemeal. So, for instance, ```DM::new()``` should definitely just be unwrapped.